frontend/src/app/App.tsx

```diff
@@ -207,6 +203,8 @@ export class App extends PureComponent<Props, State> {
 
   private readonly widgetMgr: WidgetStateManager
 
+  public hostCommunicationMgr: HostCommunicationManager
```

Comment:
This should be `private readonly`. (This may require some `// @ts-expect-error` in App.test.tsx, where the value is read.)